### PR TITLE
Fix sticky sessions

### PR DESCRIPTION
**Pull request by tusharshahrs**
Screen Shot of haproxy.cfg with sticky session turned ON:
tusharshah_dev-digital-deals-haproxy-test-nil-01____ _-ssh_-l_tusharshah_ec2-34-225-138-136_compute-1_amazonaws_com_ _228x39

dev-digital-deals-haproxy-test-nil-01_-_server_inputs

Screen Shot of haproxy.cfg with sticky session turned OFF:
tusharshah_dev-digital-deals-haproxy-test-nil-02____ _-ssh_-l_tusharshah_ec2-34-225-138-136_compute-1_amazonaws_com_ _241x37_and_tusharshah_dev-digital-deals-haproxy-test-nil-01____ _-ssh_-l_tusharshah_ec2-34-225-138-136_compute-1_amazonaws

dev-digital-deals-haproxy-test-nil-02_-_server_inputs

### CODE REVIEW

File: attributes/default.rb [hunk 1]
```diff
@@ -40,7 +40,7 @@
 default['rs-haproxy']['stats']['password'] = nil
 
 # Enable/Disable sticky sessions using cookie
-default['rs-haproxy']['session_stickiness'] = true
+default['rs-haproxy']['session_stickiness'] = false
```
Comment (cdwilhelm) [inline attributes/default.rb]: I believe this should be a string, 'false'  the recipe only evaluates a string, not a boolean.

Comment (tusharshahrs) [reply attributes/default.rb]: @cdwilhelm I updated it and made the change requested

**Comment by tusharshahrs** [comment]
Updated Code to use boolean instead of string

**Comment by tusharshahrs** [comment]
Here is haproxy with sticky session set to true
haproxy-test-nil-01

Here is the ServerTemplate setting set to true:
haproxy-test-nil-01-servertemplate

Here is haproxy with sticky session set to false
haproxy-test-nil-02



Here is the ServerTemplate setting set to false:
dev-digital-deals-haproxy-test-nil-02_-_server_inputs

**Pull request merged**